hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/VolumeInfoMetrics.java

```diff
@@ -104,23 +104,26 @@ public String getMetricsSourceName() {
    */
   @Metric("Returns the Used space")
   public long getUsed() {
-    return volume.getVolumeInfo().getScmUsed();
+    return volume.getVolumeInfo() != null ?
```

Comment:
getVolumeInfo() is used multiple places and its null if failedVolume. So need to have null check of isFailed check wherever this is called.
Similarly need fix for refreshVolumeInfo() also

Reply:
Hi @sumitagrawl , Thanks for the review. Handled these cases. Also getVolumeInfo() in most of the places is already checked using getFailedVolume() before usage.